### ISSUE TITLE
Text to Speech not working anymore on ThunkableX

### ISSUE THREAD

**Issue by jmcmahon-shcs** [thunkable/thunkable-issues#4]
Thanks so much for filing an issue or feature request! Please fill out the following (wherever relevant):

The more info the faster we will be able to address it!

### Platform
 ✕ iOS companion?

### Issue/Request Origin
Where did you get this issue? Email, Intercom, or Community?

### Steps to Reproduce
1.  Drag Text to speech component into non-visible components
2. Add block to tell it to speak
3.  Add text into the block
4. Test and nothing happens

### Expected Behavior
Speak the text in the text block

### Actual Behavior
What actually happens?
Nothing

STACK TRACE AND/OR SCREENSHOTS
![screen shot 2018-06-17 at 12 16 59 pm](https://user-images.githubusercontent.com/40327294/41503437-653ef35c-7228-11e8-8ccd-df5a1255e645.png)

  
```

**Comment by mrching** [comment]
I just tried this on several apps including the Feelings sample app and it works on iOS.  I'm wondering if you have your sound turned off on your phone.  It's the hardware switch on the left side of most iPhones.

**Comment by jmcmahon-shcs** [comment]
We have tried the hardware solution and that is not the issue.  I am doing this with a class and we all get the same result (and all checked volume and ringer settings).  It will now work if we plug in directly a text block and call the speak when it opens the screen.  However, on subsequent screens if we call speak by plugging in the value of a variable or the text from a label, nothing happens at all.  I have included a screen shot of the code here.
